video/avi_decoder.cpp

```diff
@@ -1038,13 +1035,12 @@ void AVIDecoder::AVIVideoTrack::setDither(const byte *palette) {
 	_videoCodec->setDither(Image::Codec::kDitherTypeVFW, palette);
 }
 
-AVIDecoder::AVIAudioTrack::AVIAudioTrack(const AVIStreamHeader &streamHeader, const PCMWaveFormat &waveFormat, Audio::Mixer::SoundType soundType) :
-		AudioTrack(soundType),
-		_audsHeader(streamHeader),
-		_wvInfo(waveFormat),
-		_audioStream(0),
-		_packetStream(0),
-		_curChunk(0) {
+AVIDecoder::AVIAudioTrack::AVIAudioTrack(const AVIStreamHeader &streamHeader, const PCMWaveFormat &waveFormat, Audio::Mixer::SoundType soundType) : AudioTrack(soundType),
```

Comment:
This now looks terrible, needs to be reverted. Probably we need extending .clang-format to prevent continuation string be so extreme.

Reply:
I don't think any of the built-in formatters support the type of wrapping we'd want here